src/content/defi/index.md

```diff
@@ -24,9 +24,7 @@ DeFi is a collective term for financial products and services that are accessibl
 
 There's a booming crypto economy out there, where you can lend, borrow, long/short, earn interest, and more. Crypto-savvy Argentinians have used DeFi to escape crippling inflation. Companies have started streaming their employees their wages in real time. Some folks have even taken out and paid off loans worth millions of dollars without the need for any personal identification.
 
-<figure>
-  <iframe width="100%" height="315px" src="https://www.youtube.com/embed/H-O3r2YMWJ4" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
-</figure>
+<YouTube id="H-O3r2YMWJ4" />
```

Comment:
I like the idea here but I think only passing the `id` here adds complexity. 

- Without context, contributors might have to go to the YouTube component or read the style guide to understand how the props here work—a regular URL seems more intuitive.  
- It also seems a little finicky and prone to error to me (i.e. it is a lot easier to copy an entire URL versus only part of it.)

```suggestion
<YouTube src="https://www.youtube.com/embed/H-O3r2YMWJ4" />
```

Reply:
Agreed - @wackerow could you please update to pass the entire URL src sring?

Reply:
Same. I think once that change is in, then this is good 👍

Reply:
Main issue with this is that we use an "embed" URL:

If user grabs link from the URL bar:
![image](https://user-images.githubusercontent.com/54227730/147139697-5c108c77-6880-4baa-9f3b-9f0cf384b313.png)

If user grabs from "Share" modal:
![image](https://user-images.githubusercontent.com/54227730/147139760-4f1221bf-c94a-4811-b6b6-f6d04f496da2.png)

And if user selects "Embed":
![image](https://user-images.githubusercontent.com/54227730/147139789-baeb2c04-15c5-4ab1-b03e-df9ba83f072a.png)

In either case, none of these are exactly what we want if we're asking users to pass the entire "embed" URL into this component.

If we really don't like the idea of just using the video ID, then perhaps we should expand the logic of this component to accept any link and parse out the ID ourselves within the component. Otherwise I don't know that this will make anything any easier.

Reply:
Ah, that's a fair point!